tests/dualtor/test_standby_tor_upstream_mux_toggle.py

```diff
@@ -71,6 +74,9 @@ def test_standby_tor_upstream_mux_toggle(
     set_mux_state(rand_selected_dut, tbinfo, 'standby', [itfs], toggle_all_simulator_ports)     # noqa F405
     # Wait sometime for mux toggle
     time.sleep(PAUSE_TIME)
+    # Verify packets are not go up
+    crm_facts1 = rand_selected_dut.get_crm_facts()
```

Comment:
same as above

Reply:
Followed the same solution/steps as above.